src/redux-query-sync.js

```diff
@@ -71,7 +71,7 @@ function ReduxQuerySync({
         // For each parameter value that changed, call the corresponding action.
         Object.keys(queryValues).forEach(param => {
             const value = queryValues[param]
-            if (value !== lastQueryValues[param]) {
+            if (value !== lastQueryValues[param] && value !== undefined) {
```

Comment:
Actually I need to better understand what is going on, I see an unexpected behaviour(`initialTruth: 'location'` and `defaultValue: undefined`): the user can repeatedly click on the browser back arrow and it remains active.

I think this is also an issue to better investigate and I keep it off this PR.